font-stix-otf-arm.yaml

```diff
@@ -0,0 +1,37 @@
+# 2.13b171 is the latest version in github releases: https://github.com/stipub/stixfonts/releases
+package:
+  name: font-stix-otf-arm
+  version: 2.13
+  epoch: 0
+  description: stix-otf fonts
+  copyright:
+    - license: OFL-1.1
+  target-architecture:
+    - aarch64
+  dependencies:
+    provides:
+      - font-stix-otf=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - build-base
```

Comment:
```suggestion

```
nit: `build-base` is required only when we're compiling something, in this case only busybox will be good.